WcaOnRails/app/views/competitions/_nav.html.erb

```diff
@@ -135,20 +135,20 @@
       <h3><%= @competition.name %></h3>
       <hr />
 
-      <% if !@competition.showAtAll? %>
-        <div class="alert alert-warning">
-          <strong>Note:</strong> This competition is not visible to the public.
-        </div>
-      <% end %>
-      <% if !@competition.results_uploaded? %>
-        <% if @competition.is_over? %>
-          <div class="alert alert-info">
-            <strong>Note:</strong> This competition is over, we are working to upload the results as soon as possible!
+      <% unless @competition.warnings.empty? %>
```

Comment:
Is there any value to checking if it's empty before going into the loop? I feel like we're just going to end up calling the method twice sometimes.